pkgs/by-name/le/lean4/package.nix

```
@@ -8,29 +8,46 @@
   cadical,
   pkg-config,
   libuv,
+  use-mimalloc ? true,
```

```suggestion
  enableMimalloc ? true,
```

(standard naming of configure-like options)